data/templates/nginx/plain/acme-challenge.conf.inc

```diff
@@ -0,0 +1,5 @@
+location ^~ '/.well-known/acme-challenge/'
```

Comment:
When is this file copied to the right place during the installation process?

Reply:
https://github.com/YunoHost/yunohost/blob/unstable/data/hooks/conf_regen/15-nginx#L24
called by https://github.com/YunoHost/yunohost/blob/unstable/data/hooks/conf_regen/15-nginx#L11
itself called by https://github.com/YunoHost/yunohost/blob/unstable/debian/postinst#L12

(though maybe the tests disagree :D )

Reply:
(Uh I linked that from unstable instead of stretch-unstable for some reason ... but shouldnt have changed since)

Reply:
The postinstall includes a regen-conf for nginx though so it should all be consistent if you at least re-run the postinstall ?

Reply:
Yeah, I just saw that! All right then!